xc-admin/packages/xc-admin-common/src/multisig_transaction/WormholeInstruction.ts

```diff
@@ -0,0 +1,50 @@
+import { createReadOnlyWormholeProgramInterface } from "@certusone/wormhole-sdk/lib/cjs/solana/wormhole";
+import { WormholeInstructionCoder } from "@certusone/wormhole-sdk/lib/cjs/solana/wormhole/coder/instruction";
+import { getPythClusterApiUrl } from "@pythnetwork/client/lib/cluster";
+import { Connection, TransactionInstruction } from "@solana/web3.js";
+import { MultisigInstruction } from ".";
+import { decodeGovernancePayload } from "../governance_payload";
+import { AnchorAccounts, resolveAccountNames } from "./anchor";
+
+export class WormholeInstruction implements MultisigInstruction {
```

Comment:
I prefer to name it `WormholeMultisigInstruction` to distinguish it with other wormhole related things.